README.md

```diff
@@ -121,8 +122,70 @@ service](https://console.redhat.com/openshift/create).
    ```
 
 
-1. Check all applications are synchronised
+1. Check all applications are synchronized
 
+# Factory (Edge) setup
+
+It's time to connect a "remote" factory cluster to the data center (hub).
+
+## Factory setup using the ACM UI
```

Comment:
We might want to describe how to get the route for the ACM UI and how to log in (i.e., using the kubeadmin-password for hub)